codex-rs/tui/src/slash_command.rs

```diff
@@ -15,17 +15,13 @@ pub enum SlashCommand {
     New,
     Diff,
     Quit,
-    ToggleMouseMode,
 }
 
 impl SlashCommand {
     /// User-visible description shown in the popup.
     pub fn description(self) -> &'static str {
         match self {
             SlashCommand::New => "Start a new chat.",
-            SlashCommand::ToggleMouseMode => {
```

Comment:
Nice! We can also remove `disable_mouse_capture` in the docs (`config.md`) and `Config` code.